### PR TITLE
Revert "Add checkbox to optionally remove all test results aka. repor…

### PR DESCRIPTION
**Pull request by dLobatog**
…ts (#392)"

This reverts commit 2571724bda570c7d3e13be7addf00dcdcb2261e0.

Since soft-deleting policies is something we're not doing for our current goal, we are now just deleting policies *and* test results always. I reverted the commit and included 'deleteAllTestResults: true'.

### CODE REVIEW

File: src/SmartComponents/DeletePolicy/DeletePolicy.js [hunk 1]
```diff
@@ -50,8 +43,7 @@ const DeletePolicy = ({ isModalOpen, policy, toggle, onDelete }) => {
                 <Button key='destroy'
                     aria-label="delete"
                     variant='danger'
-                    onClick={() => deletePolicy({ variables: { input: { id, deleteAllTestResults } } })}
-                >
+                    onClick={() => deletePolicy({ variables: { input: { id, deleteAllTestResults: true } } })}>
```
Comment (bastilian) [inline src/SmartComponents/DeletePolicy/DeletePolicy.js:46]: Should we revert the change in the backend that went along with this and have this behaviour by default in the backend?

Comment (dLobatog) [reply src/SmartComponents/DeletePolicy/DeletePolicy.js:46]: Agreed, I'll do that - we can always get it back by re-reverting
